docs/antipatterns/optional-attributes/index.md

```diff
@@ -48,7 +48,7 @@ Suppose then that you come up with this solution:
 from enum import Enum
 from collections.abc import Mapping
 from dataclasses import dataclass
-
+from typing import Optional
```

Comment:
Maybe it would be better to replace `Optional` with `| None` in this article? It's kinda soft-deprecated